repository/src/raw/reserved_state.rs

```diff
@@ -86,49 +86,7 @@ mod tests {
 
     #[tokio::test]
     async fn format_reserved_state() {
-        let genesis_info = GenesisInfo {
-            header: BlockHeader {
-                height: 3,
-                author: PublicKey::zero(),
-                prev_block_finalization_proof: vec![TypedSignature::new(
-                    Signature::zero(),
-                    PublicKey::zero(),
-                )],
-                previous_hash: Hash256::hash("hello1"),
-                timestamp: 0,
-                commit_merkle_root: Hash256::hash("hello2"),
-                repository_merkle_root: Hash256::hash("hello3"),
-                validator_set: vec![(PublicKey::zero(), 1)],
-                version: "0.1.0".to_string(),
-            },
-            genesis_proof: vec![TypedSignature::new(Signature::zero(), PublicKey::zero())],
-            chain_name: "chain".to_string(),
-        };
-
-        let member1 = Member {
-            public_key: PublicKey::zero(),
-            name: "name1".to_string(),
-            governance_voting_power: 1,
-            consensus_voting_power: 1,
-            governance_delegations: Some(PublicKey::zero()),
-            consensus_delegations: Some(PublicKey::zero()),
-        };
-        let member2 = Member {
-            public_key: PublicKey::zero(),
-            name: "name2".to_string(),
-            governance_voting_power: 1,
-            consensus_voting_power: 1,
-            governance_delegations: Some(PublicKey::zero()),
-            consensus_delegations: Some(PublicKey::zero()),
-        };
-        let members: Vec<Member> = vec![member1, member2];
-
-        let reserved_state = ReservedState {
-            genesis_info,
-            members,
-            consensus_leader_order: vec![0, 1],
-            version: "0.1.0".to_string(),
-        };
+        let (reserved_state, _) = simperby_test_suite::generate_standard_genesis(10);
```

Comment:
some `rs`, some `reserved_state`

not necessary but I think consistent naming is good

Reply:
They're already interchangeably used in the project. This single PR can't fix all.

Reply:
@junha1 let's just do all those kinds of stuffs after the genesis